PWGHF/Core/SelectorCuts.h

```diff
@@ -94,6 +94,13 @@ static const std::vector<std::string> labelsPt{};
 
 // column labels
 static const std::vector<std::string> labelsCutBdt = {"BDTbkg", "BDTprompt", "BDTnonprompt"};
+
+// ONNX file names
+static const std::string onnxFileNameSpecies[5][1] = {{"ModelHandler_onnx_D0ToKPi.onnx"}, {"ModelHandler_onnx_DplusToPiKPi.onnx"}, {"ModelHandler_onnx_LcToPKPi.onnx"}, {"ModelHandler_onnx_DsToKKPi.onnx"}, {""}};
```

Comment:
@vkucera Megalinter fails because it wants this to be `static const char*` instead of `static const std::string`, but it does not compile with `char` instead of `std::string`. Any idea how to fix it?

Reply:
What is the error?
`static const char* onnxFileNameSpecies[5][1] = ...` compiles just fine.

Reply:
Did you try? It does not compile for me (full log attached):
```
/home/fgrosa/alice/sw/SOURCES/O2Physics/dev_trigger/0/PWGHF/TableProducer/trackIndexSkimCreator.cxx:1227:234: error: could not convert '{"onnxFileNames", {o2::analysis::hf_cuts_bdt_multiclass::onnxFileNameSpecies[0], 5, 1, o2::analysis::hf_cuts_bdt_multiclass::labelsSpecies, o2::analysis::hf_cuts_bdt_multiclass::labelsPt}, "ONNX file names for ML models"}' from '<brace-enclosed initializer list>' to 'o2::framework::Configurable<o2::framework::LabeledArray<std::__cxx11::basic_string<char> > >'
 1227 |   Configurable<LabeledArray<std::string>> onnxFileNames{"onnxFileNames", {hf_cuts_bdt_multiclass::onnxFileNameSpecies[0], 5, 1, hf_cuts_bdt_multiclass::labelsSpecies, hf_cuts_bdt_multiclass::labelsPt}, "ONNX file names for ML models"};
      |                                                                                                                                                                                                                                          ^
      |                                                                                                                                                                                                                                          |
      |                                                                                                                                                                                                                                          <brace-enclosed initializer list>
```
[log.txt](https://github.com/AliceO2Group/O2Physics/files/14847459/log.txt)

Reply:
I can confirm seeing the same errors as you do.
There are 2 other files in O2Physics with a `Configurable<LabeledArray<std::string>>` and, although the MegaLinter logs are no longer available for the PRs that introduced those configurables, I can see that the checks failed as well, so I don't have an example that would pass.
In this case, I admit I don't understand what cpplint considers wrong in your implementation so I would be fine with merging it regardless.
On the other hand, it would be good to understand why the conversion from `static const char*` fails. Maybe @aalkin could clarify whether this is expected.

Reply:
Hi @vkucera many thanks for checking and confirming! Megalinter would not complain if the `string` is not global/static, so if I define it directly in the task, but I don't particularly like this solution.

Reply:
I'm a bit late for the party, but for the static strings consider using `std::string_view` instead of `std::string` that is allocatable and thus cannot really be static. Labeled array with strings is special, since string is not a simple type and the constructor therefore expects an array of string objects or an array of something that can be converted to string, but since `const char*` is an "array" in itself, its interpretation is ambiguous.